prow/config/jobs/envoy.yaml

```diff
@@ -38,23 +38,6 @@ jobs:
     value: "/home/prow/go/src/istio.io/envoy"
   command: [./ci/do_ci.sh, bazel.release]
 
-- name: update-proxy
```

Comment:
Does generate.go rely on this? Do we need to do something fancy so it gets added to to the branches but not run on master?

Reply:
This job was just moved from being run on the envoy repos to being run on the proxy repos. I do see I forgot to remove the one periodic from the generated envoy-1.0 so new commit coming for that.

The generator.go should copy the job from the proxy master to the new proxy-{release} yamls.

The one missing thing for the instructions that generate.go will use the `main` envoyproxy branch when the new release branches are cut. The UPDATE_BRANCH will need to be updated in the yaml, or it can be updated in the istio/proxy/scripts/update_envoy.sh from `main` to the correct branch once it's released.